Source/Details/ASCollectionFlowLayoutDelegate.m

```diff
@@ -70,6 +70,7 @@ + (ASCollectionLayoutState *)calculateLayoutWithContext:(ASCollectionLayoutConte
                                                                       alignItems:ASStackLayoutAlignItemsStart
                                                                         flexWrap:ASStackLayoutFlexWrapWrap
                                                                     alignContent:ASStackLayoutAlignContentStart
+                                                                     lineSpacing:0
```

Comment:
Since we have a convenient initializer that doesn't require `lineSpacing`, please remove this change.